export.Dockerfile

```diff
@@ -57,6 +58,7 @@ ENV POSTGRES_JAR_VERSION="42.2.9"
 ENV POSTGRES_JAR_URL="https://jdbc.postgresql.org/download/postgresql-${POSTGRES_JAR_VERSION}.jar" \
     POSTGRES_JAR_PATH=$SQOOP_HOME/lib/postgresql-${POSTGRES_JAR_VERSION}.jar \
     JAVA_HOME="/usr/lib/jvm/java-11-amazon-corretto"
+    #JAVA_HOME="/usr/lib/jvm/java-11-amazon-corretto.aarch64"
```

Comment:
JAVA_HOME may have to be adapted to the value below, not sure if it was due to installing R or something to do with building it locally leaving it as the old value for now.